nlesc-smcm/i-emic
# Move global grid initialisation to C++

Following steps to check nothing breaks:
 - Adds accessors for Fortran arrays.
 - Compare TRIOS_Domain position methods with Fortran initialised values.
 - Add tests for TRIOS_Domain to compare against references so future commits can't break the logic.
 - Drop Fortran initialisation (this should cause a test to fail, but doesn't)
 - Enable C++ initialisation.
